gs_chunked_io/writer.py

```diff
@@ -35,6 +41,13 @@ def __init__(self,
         self._current_part_number = 0
         self._closed = False
         self._upload_id = uuid.uuid4()
+        if threads is not None:
+            assert 2 <= threads
```

Comment:
Why not let `threads=1` be the same as `threads=None` and allow it to run?  I'd expect `threads=1` to work and be a viable call.

Reply:
I suppose `threads=1` should be allowed since that will use the async framework, and reads will be concurrent with the main thread.

`threads=None` should also be allowed to avoid concurrency altogether.

Reply:
will fix.